db/user_test.go

```diff
@@ -14,6 +15,46 @@ type SyncNewUsersTestCase struct {
     sendEmails bool
 }
 
+func (this *DBTests) DBTestResolveUsers(test *testing.T) {
+    defer ResetForTesting();
+
+    testCases := []struct{input []string; expectedOutput []string; course *model.Course; addUsers []*model.User; removeUsers []string} {
+        {[]string{""}, make([]string, 0, 1), nil, nil, make([]string, 0, 1)},
```

Comment:
Don't use make like this in test cases.
Just use a literal slice definition (like you did with `input`).